package.json

```diff
@@ -83,8 +83,7 @@
     "which": "^2.0.2",
     "xterm": "^4.9.0",
     "xterm-addon-fit": "^0.5.0",
-    "xterm-addon-search": "^0.8.0",
-    "yarn-or-npm": "^3.0.1"
+    "xterm-addon-search": "^0.8.0"
```

Comment:
note: I think `detect-package-manager` should be added to the top-level `package.json` as well.